changelog/fragments/add-componentconfig-flag.yaml

```diff
@@ -0,0 +1,26 @@
+entries:
+  - description: >
+      (ansible/v1, helm/v1) correctly support configuring the manager by file using the
+      `--config` flag. This flag was not added to either ansible-/helm-operator binary
+      when file support was originally added.
+    kind: bugfix
+    migration:
+      header: Add the manager config patch to config/default/kustomization.yaml
+      body: >
+        The scaffolded `--config` flag was not added to either ansible-/helm-operator binary
+        when config file support was originally added, so does not currently work.
+        The `--config` flag was added to both binaries to support configuration by file;
+        this method of configuration only applies to the underlying
+        [controller manager](https://pkg.go.dev/sigs.k8s.io/controller-runtime/pkg/manager#Manager),
+        not the operator as a whole. To configure the operator's Deployment with this config data,
+        add the following to config/default/kustomization.yaml:
+
+        ```diff
+        # If you want your controller-manager to expose the /metrics
+        # endpoint w/o any authn/z, please comment the following line.
+        - manager_auth_proxy_patch.yaml
+        +
+        +# Mount the controller config file for loading manager configurations
+        +# through a ComponentConfig type
+        +#- manager_config_patch.yaml
```

Comment:
Are we not here to explain how to use the feature? So, should that be added here or in a doc? Maybe FAQ if a question was raised about that where we could add the steps + link the doc: https://master.book.kubebuilder.io/component-config-tutorial/tutorial.html . WDYT?

Reply:
Added the link. That should be sufficient for those curious, especially since this feature is optional and opt-in.